crates/core_arch/src/x86_64/amx.rs

```diff
@@ -0,0 +1,563 @@
+/// Load tile configuration from a 64-byte memory location specified by mem_addr.
+/// The tile configuration format is specified below, and includes the tile type pallette,
+/// the number of bytes per row, and the number of rows. If the specified pallette_id is zero,
+/// that signifies the init state for both the tile config and the tile data, and the tiles are zeroed.
+/// Any invalid configurations will result in #GP fault.
+///
+/// [Intel's documentation](https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_tile_loadconfig&ig_expand=6875)
+#[inline]
+#[target_feature(enable = "amx-tile")]
+#[unstable(feature = "x86_amx_intrinsics", issue = "126622")]
+pub unsafe fn _tile_loadconfig(mem_addr: *const i8) {
```

Comment:
For void pointers in Rust we generally use `*mut u8`. This is what the allocator API uses for example. At the LLVM level there is no distinction between `i8` and `u8` which is why LLVM intrinsics are always show taking `i8`.

Reply:
make sense, let's make it more idiomatic